db/schema.rb

```diff
@@ -1336,7 +1336,6 @@
 
   create_table "users_settings", force: :cascade do |t|
     t.string "brand_color1", default: "#000000"
-    t.string "brand_color2", default: "#ffffff"
```

Comment:
Add this field back for now. The proper way to remove a field is to create a [rails migration](https://guides.rubyonrails.org/active_record_migrations.html#creating-a-standalone-migration) and you want to do this in a separate PR after all the use of `brand_color2` has been removed. To aid you in this change you should also use [ignored_columns](https://www.bigbinary.com/blog/rails-5-adds-active-record-ignored-columns). I can go into more details if you would like me to

Reply:
Reverted this change.

Reply:
I will do this change in a separate PR.